### PR TITLE
Feat/frontend mbr gbr foolproof mechanism

### PR DESCRIPTION
**Pull request by Alonza0314**
Add friendly alert for MBR should be greater than GBR(both subscription and profile).

MBRGBRgreateralert

### CODE REVIEW

File: frontend/src/pages/ProfileCreate/index.tsx [hunk 1]
```diff
@@ -11,6 +11,30 @@ import ProfileFormBasic from "./ProfileFormBasic";
 import ProfileFormUeAmbr from "./ProfileFormUeAmbr";
 import ProfileFormSessions from "./ProfileFormSessions";
 import { ProfileMapperImpl, FlowsMapperImpl } from "../../lib/dtos/profile";
+
+function parseDataRate(rate: string | undefined): number {
```
[the remark this answers is not in the document]
Comment (Alonza0314) [reply frontend/src/pages/ProfileCreate/index.tsx]: OK! Moved!

**Comment by Alonza0314** [comment]
> Hi @Alonza0314 ,
> 
> I am unfamiliar with the `zod` package, but I think there is a way to add the validation in the type schema instead of creating another validation code.
> 
> Maybe you can try `zod refine` in Google search(? ( I'm not sure XD)

I've checked the DTO schema which all of them are written via `zod`. I think we can `feat` this function first and make the validation in another PR for the whole DTO schemas(profile and subscriber) if needed(?

**Comment by Alonza0314** [comment]
> Hi @Alonza0314 ,
> 
> I am unfamiliar with the `zod` package, but I think there is a way to add the validation in the type schema instead of creating another validation code.
> 
> Maybe you can try `zod refine` in Google search(? ( I'm not sure XD)

Hi, @andy89923,
I discussed this with Ian yesterday. For now, we can proceed like this and add the valid part later in the backend.

File: frontend/src/pages/SubscriberCreate/index.tsx [hunk 2]
```diff
@@ -127,6 +147,24 @@ function SubscriberCreate() {
     const subscriberMapper = new SubscriptionMapperImpl(new SubscriptionFlowsMapperImpl());
     const subscription = subscriberMapper.mapFromDto(data);
 
+    for (let i = 0; i < subscription.QosFlows.length; i++) {
```
[the remark this answers is not in the document]
Comment (Alonza0314) [reply frontend/src/pages/SubscriberCreate/index.tsx]: Refactored!

**Review by andy89923 — APPROVED** [review]
LGTM

**Pull request merged**